keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -290,6 +290,27 @@ type IngressBackendHTTPHealthcheck struct {
   // FailureThreshold is the number of consecutive failures necesseary to
   // indicate a backend failure.
   FailureThreshold int
+  // TLS settings to use when healthchecking.
+  // +optional
+  TLS *IngressBackendHTTPHealthcheckTLS
+}
+
+// IngressBackendHTTPHealthcheckTLS is TLS settings for HTTP healthchecking.
+type IngressBackendHTTPHealthcheckTLS struct {
```

Comment:
With the caveat that a KEP is not an API review...

I am not sure all the implementations can support most of these features.  I know Google's LB can support HTTPS but does only offers (I think) unverified, and can not specify a CA or anything more.

From what I can see Amazon LB is the same.

Reply:
I agree that the API is awkward, but the intention is to place the framework for typical HTTPS backend connections.